types/index.d.ts

```diff
@@ -0,0 +1,15 @@
+import * as DRange from "drange";
+
+declare namespace RandExp {}
+
+declare class RandExp {
+    static randexp(pattern: string | RegExp, flags?: string): string;
+    static sugar(): void;
+    constructor(pattern: string | RegExp, flags?: string);
+    gen(): string;
+    defaultRange: DRange;
+    randInt: (from: number, to: number) => number;
+    max: number;
```

Comment:
Not sure about the inconsistent indent in these files. Some use 4 spaces, others 2, the project uses 2. Is using 4 the default in typescript?

Reply:
In TypeScript, 4 spaces is used in docs/generated codes. Using 4 seems to be default in TypeScript files.